src/ORM/DatabaseAdmin.php

```diff
@@ -122,8 +124,11 @@ public function build()
         // The default time limit of 30 seconds is normally not enough
         Environment::increaseTimeLimitTo(600);
 
-        // Get all our classes
-        ClassLoader::inst()->getManifest()->regenerate(false);
+        // Flush the manifest if we haven't already
+        $kernel = Injector::inst()->get(Kernel::class);
+        if (!$kernel->getFlushedOnBoot()) {
+            ClassLoader::inst()->getManifest()->regenerate(false);
+        }
```

Comment:
My intention was to store whether a flush has been performed in the kernel, and then look to refactor other places in core where we have hardcoded URLs/query strings to fetch that info from the kernel instead. Not necessary for this PR though, removed it